WatchParty/backend/models/sublist.js

```diff
@@ -0,0 +1,33 @@
+module.exports = (sequelize, DataTypes) => {
+  const Sublist = sequelize.define("Sublist", {
+    name: {
+		type: DataTypes.STRING(32),
+		allowNull: false,
+		unique: 'uniquecombination',
+		validate: { notNull: true }
+    },
+    userid: {
+		type: DataTypes.INTEGER,
+		allowNull: false,
+		unique:'uniquecombination',
+		validate: { notNull: true }
+    },
+    movie: {
+		type: DataTypes.INTEGER,
+		allowNull: false,
+		validate: { notNull: true }
+    }
+},{
+    freezeTableName: true
+	});
+	Sublist.associate = function(models) {
+	    Sublist.belongsTo(models.User,{
+		foreignKey: 'userid',
+		onDelete: 'cascade'
+	});
+	Sublist.belongsTo(models.Movie,{
```

Comment:
This is a belongs-to-many relation (we have 1 sublist for many movies https://sequelize.readthedocs.io/en/latest/docs/associations/#one-to-many-associations)

Reply:
Added has many to user and movie